ui/packages/shared/profile/src/ProfileExplorer/index.tsx

```diff
@@ -126,6 +134,14 @@ const ProfileExplorerApp = ({
     return selectProfile(p, '_b');
   };
 
+  if (profileTypesLoading) {
+    return <>{loader}</>;
+  }
+
+  if (profileTypesData?.types.length === 1) {
```

Comment:
just to confirm, do we only want to show the no data prompt when the length === 1? Would `profileTypesData?.types.length` ever return 0 or -1?

Reply:
My bad, was using 1 for testing and pushing that too. 😐  
Fixing it now.